src/plugin/sdl/sdl.c

```diff
@@ -440,33 +490,25 @@ static void *render_thread(void *arg)
 int SDL_set_videomode(struct vid_mode_params vmp)
 {
   v_printf
-      ("SDL: X_setmode: video_mode 0x%x (%s), size %d x %d (%d x %d pixel)\n",
+      ("SDL: set_videomode: 0x%x (%s), size %d x %d (%d x %d pixel)\n",
        video_mode, vmp.mode_class ? "GRAPH" : "TEXT",
        vmp.text_width, vmp.text_height, vmp.x_res, vmp.y_res);
   if (win_width == vmp.x_res && win_height == vmp.y_res) {
     v_printf("SDL: same mode, not changing\n");
     return 1;
   }
-  if (vmp.mode_class == TEXT && !use_bitmap_font)
-    SDL_change_mode(0, 0, vmp.text_width * font_width,
-		      vmp.text_height * font_height);
+  if (vmp.mode_class == TEXT)
```

Comment:
Why to remove `!use_bitmap_font`?
Bitmap font is a VGA rom font, not BDF font.

Reply:
In X plugin use_bitmap_font was set only if the X_font.c didn't like the font it was given. I'm currently causing dosemu to fallback to X plugin if we don't get an acceptable font for sdl. Is that not an appropriate course of action, although I can reinstate if you like?

Reply:
Oh yes I forgot that `use_bitmap_font` was also set in the case that SDL wasn't running under X.

Reply:
> In X plugin use_bitmap_font was set only if the X_font.c didn't like the font it was given.

Not only that.
`_X_load_text_font()` has `if (p && *p) {`
which means `use_bitmap_font` would be set
also if `config.X_font` is not set. Which is the main
point. But the X plugin code is quite bad so it hides
it.

Reply:
Of course it should have been written differently:
```
if (config.X_font && config.X_font[0])
  use_bitmap_font = !X_load_text_font();
else
  use_bitmap_font = 1;
```
But this is how it works, anyway.

Reply:
So `use_bitmap_font` is almost always 1, as
we have the VGA rom fonts by default. Its not
an indicator of the failed font load.

Reply:
Replaced `use_bitmap_font`